docs/guides/dynamo_run.md

```diff
@@ -454,7 +457,12 @@ See instructions [here](https://github.com/ai-dynamo/dynamo/blob/main/examples/t
 
 Execute the following to load the TensorRT-LLM model specified in the configuration.
 ```
-dynamo run out=pystr:/workspace/examples/tensorrt_llm/engines/trtllm_engine.py  -- --engine_args /workspace/examples/tensorrt_llm/configs/llm_api_config.yaml
+dynamo-run in=http out=trtllm TinyLlama/TinyLlama-1.1B-Chat-v1.0
+```
+
+If using KV routing, you should also enable kv cache event and metrics publishers like below:
+```
+dynamo-run in=http out=trtllm TinyLlama/TinyLlama-1.1B-Chat-v1.0 -- --publish-events-and-metrics
```

Comment:
Passing extra args after ` -- ` only worked for build-your-own-engine.

Reply:
Removing this documentation for now. Will enable this when both this and pystr and pytok removal PR is merged.